uniffi_bindgen/src/bindings/swift/mod.rs

```diff
@@ -66,15 +67,10 @@ impl BindingGenerator for SwiftBindingGenerator {
 
     fn update_component_configs(
         &self,
-        settings: &GenerationSettings,
+        _settings: &GenerationSettings,
```

Comment:
I personally would rather see changing the trait to `Option<&GenerationSettings>` instead.

Reply:
This is implemented for all the bindings though.  Some bindings, like Python, currently need the `GenerationSettings` to be passed.

Reply:
Yeah, we would need to change a lot of files.